com.unity.ml-agents/Runtime/Agent.cs

```diff
@@ -568,10 +580,16 @@ public virtual void CollectObservations(VectorSensor sensor)
         }
 
         /// <summary>
-        /// Collects the vector observations of the agent.
+        /// Collects the vector observations of the agent alongside the masked actions.
         /// The agent observation describes the current environment from the
         /// perspective of the agent.
         /// </summary>
+        /// <param name="sensor">
+        /// The vector observations for the agent.
+        /// </param>
+        /// <param name="actionMasker">
+        /// The masked actions for the agent.
+        /// </param>
         /// <remarks>
```

Comment:
Can we dedupe these comments between the overloaded methods? Seems like these will get out of sync.

Reply:
Great point. I'll include that in a follow-up PR.